x-pack/plugins/apm/ftr_e2e/cypress/support/commands.ts

```diff
@@ -78,3 +83,21 @@ Cypress.Commands.add(
     });
   }
 );
+
+// A11y configuration
+
+const axeConfig = {
+  ...AXE_CONFIG,
+  rules: [...AXE_CONFIG.rules],
+};
+const axeOptions = {
+  ...AXE_OPTIONS,
+  runOnly: [...AXE_OPTIONS.runOnly, 'best-practice'],
+};
+
+export const checkA11y = (skipFailures: boolean) => {
+  cy.injectAxe();
+  cy.configureAxe(axeConfig);
+  const context = '.kbnAppWrapper'; // Scopes a11y checks to only our app
+  cy.checkA11y(context, axeOptions, undefined, skipFailures);
```

Comment:
What's `undefined` ? 

I'd prefer this:
```suggestion
  cy.checkA11y( { context, axeOptions, skipFailures });
```

But I assume that's out of our control.

Reply:
I can't change that unfortunately

Reply:
@sqren The `undefined` here is a placeholder for an optional violations callback function that ultimately would be passed to the axe-core lib. The cypress-axe docs do a good job unpacking the arguments if you're curious: https://www.npmjs.com/package/cypress-axe#user-content-parameters-on-cychecka11y-axerun